README.md

```diff
@@ -221,3 +224,81 @@ Depending on the input parameters, the SWS may decide one of the following:
 - `HUMAN_ON_THE_LOOP`: "Inform supervisor, who may potentially intervene"
 - `HUMAN_IN_THE_LOOP`: "Let the worker decide"
 - `HUMAN_MANUALLY`: "Let the worker carry the box"
+
+___
+
+### Construction Scenario
+
+This example scenario is from the construction domain:
+
+- imagine a robot doing work on a construction site
+- while moving, the robot detects an unexpected obstacle in its way (e.g. some construction materials that are stored there temporarily)
+- depending on the situation, either of the following may now happen:
+  - the robot may be able to autonomously circumnavigate the obstacle and continue its work
+  - the robot may be blocked and thus unable to continue its work, so that it requires human support
+- depending on the situation, SWS should support the decision if/when a human should be called for help.
+
+The SWS management should decide if/when a human should be called to help the robot. This decision depends on:
+
+- the time that the robot is already blocked
+- the battery status of the robot
+- the expected waiting time until the human will come to help
+
+#### Example Rules
+
+- if the battery status of robot is low, ask for human help
+- if the time of robot blocked is short and battery status of robot is not low, let robot continue trying
+- if the time of robot blocked is moderate and battery status of robot is not low, Inform human about the problem
```

Comment:
```suggestion
- if the time the robot is already blocked is moderate and the battery status of the robot is not low, inform the human about the problem (so that they may decide if/when to help)
```